packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -1,9 +1,22 @@
-import { html } from '@microsoft/fast-element';
+import { html, when } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
 import type { TableHeader } from '.';
+import { IconArrowDown } from '../../../icons/arrow-down';
+import { IconArrowUp } from '../../../icons/arrow-up';
+import { TableColumnSortDirection } from '../../types';
 
 // prettier-ignore
 export const template = html<TableHeader>`
-    <template role="columnheader">
+    <template role="columnheader" aria-sort="${x => x.ariaSort}">
         <slot></slot>
+
+        <span class="sort-indicator">
+            ${when(x => x.sortDirection === TableColumnSortDirection.ascending, html`
```

Comment:
The icons look swapped to me; I'd expect ascending to be going up 📈

Reply:
You're absolutely right 🤦